tox.ini

```diff
@@ -1,6 +1,5 @@
 [tox]
 envlist = py27,
-          py34,
```

Comment:
keep this please, we did not drop supporting this.

Reply:
@gaborbernat are you sure about py34 support ?

Here is an extract of `setup.cfg` 

`python_requires = >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*`

Reply:
Damn 😆

Reply:
;)